### ISSUE TITLE
Admin Dashboard should provide meaningful at-a-glance information

### ISSUE THREAD

**Issue by acoffman** [genome/civic-server#36]
Recent comments are fine, but we definitely will also want things like:

* Recent evidence item submissions
* Evidence items waiting on admin acceptance
* Evidence items that have failed and require manual intervention
* Daily stats (total new users, evidence items, variants etc)

Anything else that would be useful?

**Comment by bainscou** [comment]
I think a queue of evidence items that need admin review, is the most important to have on the dashboard.

Also, it might be nice to have an easy way identify conversations that might need moderation, like a particularly active comment page, or a record that has received an above average amount of changes.

**Comment by acoffman** [comment]
The Admin dashboard will now show:

* A feed of recent events on the site (new comments, suggested changes and evidence item submissions)

* A queue of evidence items that need attention (have failed processing or are awaiting approval)

* A feed of new user signups

We can obviously add whatever else becomes useful as we continue to dogfood the site!
